Dockerfile

```
@@ -1,12 +1,17 @@
-FROM golang:1.22
+FROM golang:1.22 AS build-stage
 
-WORKDIR /app
 COPY go.mod go.sum ./
 RUN go mod download
 
 COPY . ./
 
 RUN CGO_ENABLED=0 GOOS=linux go build -o /frigabun
+
+FROM gcr.io/distroless/base-debian11 AS build-release-stage
```

```suggestion
FROM alpine
```

Universal image, even gets the whole thing down to 18.6 MB.

---

Good point, I actually meant to use the static image:
```bash
$ docker images
REPOSITORY   TAG       IMAGE ID       CREATED              SIZE
static       latest    ceb52a5217d0   About a minute ago   13.3MB
base         latest    9fc71275bcb4   About a minute ago   31.3MB
alpine       <none>    1d34ffeaf190   13 days ago          7.79MB
```